### DISCUSSION TITLE
What is the equivalent of .npmignore in deno?

### DISCUSSION THREAD

**Discussion by angelnext** [denoland/deno#15355] [Q&A]
I'm sorry if this is a stupid question, I want to release a node.js package to deno, but there are some files in my github that shouldn't be uploaded to deno.land

**Comment by zachauten** [comment]
I believe you can specify a subdirectory of your project to publish. Check out [this package's](https://deno.land/x/corejs@v3.24.1) page on deno.land/x, and compare with the actual repo. Looks like they only expose the "deno" folder.
